### PR TITLE
Run E2E tests in parallel

### PR DESCRIPTION
**Pull request by LBHCallumM**

### CODE REVIEW

File: cypress.config.js [hunk 1]
```diff
@@ -19,6 +19,7 @@ const storeData = async (data, filepath) => {
 }
 
 export default defineConfig({
+  projectId: 'fg82nr',
```
Comment (Duslerke) [inline cypress.config.js:22]: I guess that's the mfe e2e tests project id mentioned within that massive discussion thread on housing channel?

File: package.json [hunk 2]
```diff
@@ -17,9 +17,9 @@
     "lint": "eslint . && echo 'Lint complete.'",
     "lintfix": "eslint . --max-warnings 0 --fix",
     "test:e2e": "NODE_ENV=test start-test http://localhost:5001 cy:run",
-    "e2e:audit": "NODE_ENV=test start-test http://localhost:5001 cy:run:audit",
-    "cy:run": "cypress run --browser chrome --headless",
-    "cy:run:audit": "cypress run --browser chrome --headless --env runAudit=true",
+    "e2e": "NODE_ENV=test start-test http://localhost:5001 cy:run:ci",
+    "cy:run": "cypress run --browser chrome --headless --record --key 31a18a85-d0e5-4c16-a1d4-5f0eab6a7c58  --parallel --group 'all tests'",
```
Comment (Duslerke) [inline package.json:21]: Where does this `--key` value come from?

**Review by Duslerke — APPROVED** [review]

File: package.json (hunk 2 above)
Comment (LBHCallumM) [reply package.json:21]: mmh

File: cypress.config.js (hunk 1 above)
Comment (LBHCallumM) [reply cypress.config.js:22]: pretty much

**Pull request merged**